server/cli/accounting.go

```diff
@@ -105,7 +105,7 @@ func runRmAcct(cmd *cobra.Command, args []string) {
 // A setAcctCmd command creates a new or updates an existing acct
 // config.
 var setAcctCmd = &cobra.Command{
-	Use:   "set-acct [options] <key-prefix> <acct-config-file>",
+	Use:   "set [options] <key-prefix> <acct-config-file>",
 	Short: "create or update an accounting config for key prefix\n",
```

Comment:
FYI, we were using `\n` at the end of the `Short` string to create groupings. I think these can be removed now.